packages/cdktf/lib/terraform-variable.ts

```diff
@@ -51,6 +51,11 @@ export abstract class VariableType {
   }
 }
 
+export interface TerraformVariableValidationConfig {
+  readonly condition: any;
+  readonly error_message: string;
```

Comment:
I think we'd want to change this to `errorMessage` to be consistent with other parts of our codebase.
This would then need some mapping in `synthesizeAttributes()` to change change it back to `error_message` for the resulting Terraform JSON.

Reply:
Done!